package.json

```diff
@@ -1,565 +1,577 @@
+	"publisher": "vsmobile",
+	"icon": "images/icon.png",
+	"galleryBanner": {
+		"color": "#3B3738",
+		"theme": "dark"
+	},
+	"description": "Code-hinting, debugging and integrated commands for React Native",
+	"keywords": [
+		"react-native",
+		"multi-root ready"
+	],
+	"bugs": "https://github.com/Microsoft/vscode-react-native/issues",
+	"license": "SEE LICENSE IN LICENSE.txt",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/Microsoft/vscode-react-native"
+	},
+	"engines": {
+		"vscode": "^1.24.0"
+	},
+	"categories": [
+		"Debuggers",
+		"Other"
+	],
+	"activationEvents": [
+		"onDebug",
+		"onCommand:reactNative.runAndroidSimulator",
+		"onCommand:reactNative.runAndroidDevice",
+		"onCommand:reactNative.runIosSimulator",
+		"onCommand:reactNative.runIosDevice",
+		"onCommand:reactNative.runExponent",
+		"onCommand:reactNative.startPackager",
+		"onCommand:reactNative.stopPackager",
+		"onCommand:reactNative.restartPackager",
+		"onCommand:reactNative.publishToExpHost",
+		"onCommand:reactNative.showDevMenu",
+		"onCommand:reactNative.reloadApp",
+		"onCommand:reactNative.runInspector"
+	],
+	"main": "./src/extension/rn-extension",
+	"contributes": {
+		"commands": [
+			{
+				"command": "reactNative.runAndroidSimulator",
+				"title": "%reactNative.command.runAndroidSimulator.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.runAndroidDevice",
+				"title": "%reactNative.command.runAndroidDevice.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.runIosSimulator",
+				"title": "%reactNative.command.runIosSimulator.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.runIosDevice",
+				"title": "%reactNative.command.runIosDevice.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.runExponent",
+				"title": "%reactNative.command.runExponent.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.startPackager",
+				"title": "%reactNative.command.startPackager.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.stopPackager",
+				"title": "%reactNative.command.stopPackager.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.restartPackager",
+				"title": "%reactNative.command.restartPackager.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.publishToExpHost",
+				"title": "%reactNative.command.publishToExpHost.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.showDevMenu",
+				"title": "%reactNative.command.showDevMenu.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.reloadApp",
+				"title": "%reactNative.command.reloadApp.title%",
+				"category": "React Native"
+			},
+			{
+				"command": "reactNative.runInspector",
+				"title": "%reactNative.command.runInspector.title%",
+				"category": "React Native:"

```

Comment:
Could you please remove `:` in category section for consistency with other entries?